ansible/roles/atmo-user-ssh-keys/tasks/main.yml

```diff
@@ -8,24 +8,3 @@
   authorized_key: user={{ ATMOUSERNAME }} key='{{ USERSSHKEYS|join("\n") }}' state=present exclusive=yes
   when: USERSSHKEYS is defined
 
-- name: grab local copy of user ssh private key
-  set_fact: pk_file="{{ lookup('file', '/tmp/' + ATMOUSERNAME + '_default.pub') }}"
-  tags: debug
-
-- name: distribute the generated gate one key to the instance
-  authorized_key: >
-    key='{{ pk_file }}'
-    state=present
-    user="{{ item }}"
-  when: pk_file is defined
-  with_items:
-    - "{{ ATMOUSERNAME }}"
-  tags:
-    - ssh-keys
-    - debug
-
-- name: remove user key from controller node
-  local_action: file path=/tmp/{{ ATMOUSERNAME }}_default.pub state=absent
-  tags:
-    - ssh-keys
-    - delete-key
```

Comment:
This code should not be removed as this is required for passwordless login for GateOne.  Really this should be split out into different roles.